mlflow/server/js/src/components/ExperimentPage.js

```diff
@@ -81,28 +83,49 @@ class ExperimentPage extends Component {
         [props.experimentId],
         newState.persistedState.searchInput,
         lifecycleFilterToRunViewType(newState.lifecycleFilter),
+        [],
         newState.searchRunsRequestId));
       return newState;
     }
     return null;
   }
 
-  onSearch(paramKeyFilterString, metricKeyFilterString, searchInput, lifecycleFilterInput) {
+  onSearch(
+      paramKeyFilterString,
+      metricKeyFilterString,
+      searchInput,
+      lifecycleFilterInput,
+      orderByKey,
+      orderByAsc) {
     this.setState({
       persistedState: new ExperimentPagePersistedState({
         paramKeyFilterString,
         metricKeyFilterString,
         searchInput,
+        orderByKey,
+        orderByAsc,
```

Comment:
I think we unfortunately need to either replicate logic from the constructor in setting these values to non-undefined values or ensure never pass `undefined` here. For example, currently we don't pass `orderByAsc` in [onClear](https://github.com/mlflow/mlflow/pull/1455/files#diff-fa151380130110b5b1f7ba277afb356aR628) so it ends up as undefined here, which results in React warnings like:

```
Warning: Failed prop type: The prop `orderByAsc` is marked as required in `ExperimentView`, but its value is `undefined`.
    in ExperimentView (created by Connect(ExperimentView))
    in Connect(ExperimentView) (created by Route)
    in Route (created by withRouter(Connect(ExperimentView)))
    in withRouter(Connect(ExperimentView)) (at ExperimentPage.js:167)
    in RequestStateWrapper (created by Connect(RequestStateWrapper))
    in Connect(RequestStateWrapper) (at ExperimentPage.js:144)
    in div (at ExperimentPage.js:143)
    in ExperimentPage (created by Connect(ExperimentPage))
    in Connect(ExperimentPage) (created by Route)
    in Route (created by withRouter(Connect(ExperimentPage)))
    in withRouter(Connect(ExperimentPage)) (at HomeView.js:58)
    in div (at HomeView.js:56)
    in div (at HomeView.js:47)
    in HomeView (created by Connect(HomeView))
    in Connect(HomeView) (at HomePage.js:26)
    in RequestStateWrapper (created by Connect(RequestStateWrapper))
    in Connect(RequestStateWrapper) (at HomePage.js:25)
    in HomePage (created by Connect(HomePage))
    in Connect(HomePage) (created by Route)
    in Route (at App.js:55)
    in Switch (at App.js:49)
    in AppErrorBoundary (at App.js:48)
    in div (at App.js:24)
    in Router (created by HashRouter)
    in HashRouter (at App.js:23)
    in App (created by Connect(App))
    in Connect(App) (at index.js:14)
    in Provider (at index.js:13)
```